### ISSUE TITLE
Greader API: "edit-tag" method does not seem to work.

### ISSUE THREAD

**Issue by martinrotter** [FreshRSS/FreshRSS#3400]
I am the developer of RSS Guard and I am adding generic google-reader-api plugin with freshrss/bazqux/theoldreader support.

I have problem with "edit-tag" API method. I am testing the method like this:

`curl 'http://192.168.2.169/api/greader.php/reader/api/0/edit-tag?a=user/-/label/bbb&i=tag:google.com,2005:reader/item/0005ba03902249c4' -s -H 'Authorization:GoogleLogin auth=freshrss/cb82e25d91f9978153b613523f80bcd11fd2f047' -i`

I have a "bbb" tag which I add to some message. Method returns "HTTP/200/OK" code but after I refresh WEB UI, the change did not happen. Still not label assigned to the message.

One more observation is that the method does NOT in fact do input sanity check. I can input some totally made up message ID or label ID and method still returns "OK", even if those IDs do not actually exist.

**Comment by martinrotter** [comment]
Nevermind, solved it, some observations:
* There is no clear statement that "edit-tag" method uses HTTP-POST method.
* It seems that the method for me works even without "T" token.

**Comment by Frenzie** [comment]
With regard to the greader API I'll ping @Alkarex, but I would call such a distinction between GET and POST both expected and clearly stated in the HTTP specification itself. GET and "edit" are a contradiction in terms. :-)

PS Nice app.

**Comment by martinrotter** [comment]
@Frenzie Indeed, they are contradictory, but given the state of Greader API documentation, I think it would be really really nice to have complete and accurate documentation of the api somewehere as it seems that some parts are documented very vaguely.

That said, I finalized very experimental implementation of fresh rss and it seems to basically work quite nicely. RSS Guard now offers quite extensive support for FreshRSS and Bazqux and TheOldReader are on the way - I have plugin interfaces for transparent implementation of many features, including label functionality. Commit is here: https://github.com/martinrotter/rssguard/commit/6dc5e016f1f3c879ebaa6efd3886bd81a0c6c13f

After initial implementation of remaining services (which I expect to be fairly quick, because the API code is single codebase - it is in fact single RSS GUard plugin) I expect to fine tune the plugin and probably add some extra features like creating of feeds/folders/labels.

![image](https://user-images.githubusercontent.com/1255302/106273390-dff20880-6232-11eb-90dc-4a571931afad.png)
![image](https://user-images.githubusercontent.com/1255302/106273368-d23c8300-6232-11eb-817a-f5168c02bb4e.png)

**Comment by martinrotter** [comment]
@Frenzie One more thing, I did not find any forums of freshrss users, can you somehow spread the word and call people for testing? At this point it seems that RSS Guard is one of the few maintained RSS readers for Windows WITH (future) support of freshrss. The development binaries with experimental frshrss support via greader-api will be available in 15 minutes here - https://github.com/martinrotter/rssguard/releases/tag/devbuild (linux, windows, macosx) - OSS/free, portable.

**Comment by Frenzie** [comment]
@Alkarex will be able to say a lot more, but perhaps also see https://github.com/FreshRSS/FreshRSS/issues/2566#issuecomment-541317776

> I did not find any forums of freshrss users, can you somehow spread the word and call people for testing?

I mentioned it on  https://framateam.org/freshrss/pl/8h46ts6m1ff48nnpeipyojqibw

**Comment by math-GH** [comment]
Any news here?
